src/edu-benefits/schema.js

```diff
@@ -286,48 +262,58 @@ module.exports = {
       }
     },
     currentlyActiveDuty: {
-      type: 'boolean'
-    },
-    terminalLeaveBeforeDischarge: {
-      type: 'boolean'
+      type: 'object',
```

Comment:
i decided to group them together because on the question flow they are dependent questions to someone answering yes to currently active duty